tests/acceptance/features/bootstrap/FilesVersionsContext.php

```diff
@@ -52,7 +110,7 @@ class FilesVersionsContext implements Context {
 	public function userRestoresVersionIndexOfFile($user, $versionIndex, $path) {
 		$user = $this->featureContext->getActualUsername($user);
 		$fileId = $this->featureContext->getFileIdForPath($user, $path);
-		$responseXml = $this->listVersionFolder($user, "/meta/$fileId/v", 1);
+		$responseXml = $this->listVersionFolder($user, (int)$fileId, 1);
```

Comment:
```suggestion
		$responseXml = $this->listVersionFolder($user, $fileId, 1);
```

IMO file-id in OCIS is not always a numeric-string. Casting to `int` probably messes it up.